requirements/base.txt

```diff
@@ -6,7 +6,7 @@ coloredlogs==15.0
 networkx==2.6.2
 tabulate==0.8.8
 psycopg2-binary==2.9.1
-scipy==1.7.3
+scipy==1.8.1
```

Comment:
a good instruction on scipy + m1:https://stackoverflow.com/questions/65745683/how-to-install-scipy-on-apple-silicon-arm-m1